okhttp/src/test/java/io/grpc/okhttp/OkHttpClientTransportTest.java

```diff
@@ -181,16 +181,6 @@ public long read() {
     }
   }
 
-  /** Final test checks and clean up. */
-  @After
-  public void tearDown() throws Exception {
```

Comment:
Not sure what you mean by more complex.  The behavior is the same as before, it is just being called explicitly from each test.  It is a different but valid way to do it.  The name is generic because the verification is generic.  Every single test has had to have that, which means they have had to have that same processing, regardless of whether it was relevant to the test.  I looked back into the history to see when the @ After clause was added, and at the time there were only 12 tests.  That would mean the other 33 just kind had the same post processing applied.  They share the same behavior because they had to.

shutdownAndVerify is a crummy name, and is symbolic of trying to do too much.  Alternate names welcome, or better yet, splitting it apart because maybe every test doesn't need to do it.

The amount of trouble to add what amounts to 20 lines of change was far to high.  I wanted to test a single method and it required a ton of extra code.  I don't want to check if the transport is shutdown, or that there are no streams, or any of the other things the previous @ After  statement checked.  

Executor.shutdown should be in an after clause, since I agree that counts as cleanup.